.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - stable
+  - node
```

Comment:
I used `node` regarding [this](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions).